src/Compilers/VisualBasic/Portable/Binding/ImportAliasesBinder.vb

```diff
@@ -84,6 +87,37 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Return ImmutableArray(Of Symbol).Empty
             End Get
         End Property
+
+        Private ReadOnly Property ParentImportChain As IImportChain Implements IImportChain.Parent
+            Get
+                Return NextImportChain
+            End Get
+        End Property
+
+        Private ReadOnly Property Aliases As ImmutableArray(Of IAliasSymbol) Implements IImportChain.Aliases
+            Get
+                If _lazyImportChainAliases.IsDefault Then
+                    InterlockedOperations.Initialize(_lazyImportChainAliases, ComputeImportChainAliases())
+                End If
+
+                Return _lazyImportChainAliases
+            End Get
+        End Property
+
+        Private Function ComputeImportChainAliases() As ImmutableArray(Of IAliasSymbol)
+            Dim result = ArrayBuilder(Of IAliasSymbol).GetInstance(_importedAliases.Count)
+            For Each kvp In _importedAliases
+                result.Add(kvp.Value.Alias)
+            Next
+
+            Return result.ToImmutableAndFree()
+        End Function
+
+        Private ReadOnly Property [Imports] As ImmutableArray(Of INamespaceOrTypeSymbol) Implements IImportChain.Imports
+            Get
+                Return ImmutableArray(Of INamespaceOrTypeSymbol).Empty
```

Comment:
>Return ImmutableArray(Of INamespaceOrTypeSymbol).Empty [](http://example.com/codeflow?start=16&length=54)

This doesn't feel right. I think file level aliases should be grouped with types and namespaces imported on the file level

Reply:
Agreed.  done.